src/Symfony/Component/Form/DependencyInjection/FormPass.php

```diff
@@ -46,29 +50,36 @@ public function process(ContainerBuilder $container)
         }
 
         $definition = $container->getDefinition($this->formExtensionService);
```

Comment:
I just realised that I never posted my comment from January 26th on https://github.com/symfony/symfony/pull/21283 by forgetting to publish the review :(. But this is broken, we need to ensure that the defined extension is an instance of `DependencyInjectionExtension`.